bladex/propeller.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module for the propeller with shaft bottom-up parametrized construction.
+"""
+import os
+import numpy as np
+from bladex import Blade, Shaft
+import OCC.Core.TopoDS
+from OCC.Core.gp import gp_Dir, gp_Pnt, gp_Ax1, gp_Trsf
+from OCC.Core.IGESControl import IGESControl_Reader, IGESControl_Writer
+from OCC.Core.BRepBuilderAPI import BRepBuilderAPI_Transform, BRepBuilderAPI_Sewing
+from OCC.Core.BRepAlgoAPI import BRepAlgoAPI_Fuse
+from OCC.Extend.DataExchange import write_stl_file
+
+class Propeller(object):
```

Comment:
Can we add a method to plot the entire propeller?

Reply:
Thanks for the suggestion. I tried but using the init_display I had to wait a lot without success. I therefore looked at the figures opening the .stl / .iges files related to the propeller /shaft or the full body (shaft with propeller). I can give it some other try.

Reply:
Try to plot it with matplotlib as done for the blade class.

Reply:
Thank you for your suggestion. Concerning the plot method in the blade class I had difficulties also earlier (when plotting a single blade), as addressed by the issue opened by Nicola Demo.

Reply:
I think it's just a deprecated method to update.